packages/react-ui-kit/package.json

```diff
@@ -1,6 +1,28 @@
 {
   "name": "@wireapp/react-ui-kit",
-  "version": "0.0.1",
+  "version": "0.0.0",
+  "main": "./dist/index.js",
+  "repository": "https://github.com/wireapp/wire-web-packages/tree/master/packages/react-ui-kit",
+  "scripts": {
+    "clear": "rimraf dist",
+    "dist": "yarn clear && yarn babel src --out-dir dist --no-comments",
+    "fix:config": "prettier --write '**/*.json'",
+    "fix:script": "npm run test -- --fix",
+    "fix:style": "prettier --write '**/*.scss'",
+    "fix": "npm run fix:config && npm run fix:style && npm run fix:script",
+    "precommit": "lint-staged",
+    "start": "npm run start:demo",
+    "start:static": "gulp",
+    "start:demo": "webpack-dev-server",
+    "test": "eslint -c .eslintrc.yaml --ignore-path .gitignore --ignore-path .prettierignore '**/*.{js,jsx}'",
+    "preversion": "yarn test",
+    "version": "yarn dist"
+  },
+  "lint-staged": {
```

Comment:
Can be moved to the main `package.json`.

Reply:
I'm working on that one..